### DISCUSSION TITLE
how to set goals?

### DISCUSSION THREAD

**Discussion by KelvinJPS** [muety/wakapi#700] [Q&A] [answered]
this is my main feature in wakatime but I'm unable to find on wakapi

**Comment by muety** [comment, marked as answer]
Goals are currently not supported in Wakapi and likely never will be, sorry! There is #166, but, frankly, I doubt that we're going to implement it. Hope you still enjoy using Wakapi.

**Reply by KelvinJPS** [reply]
Oh what a shame, still the features that are implemented worked well, and it was super easy to install the program in my vps, well done!.
